src/js/actions/UserActions.js

```diff
@@ -43,11 +43,12 @@ export const getUserHash = () => {
     };
 };
 
-export const getUserHashSuccess = (iconHash) => {
+export const getUserHashSuccess = (icon_hash) =>{

```

Comment:
we are using camelCase for naming our variables in the front end, icon_hash and audio_name was needed to be renamed only on the requests to the backend because on the backend the variable names are written in snake_case.  can you revert the changes on the variables to keep the same notation... I do not mind to leave it like that, but I guess it is good practice to have everything with the same notation and it can avoid confusions, because right now everything that is snake_case I assume it is a request to the backend...

Reply:
that sounds good. I'll change it to camel case whenever its in the front end. thanks